BluzelleBridge/Core/Swarm.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bluzelle.NEO.Bridge.Core
+{
+    public interface ISwarm
```

Comment:
The implementation of ISwarm that connects to the DB is still TODO. Could we get that library in a separate C# language binding repo?